include/podio/Reader.h

```diff
@@ -6,6 +6,16 @@
 
 namespace podio {
 
+/// Generic (type erased) reader class that can handle different I/O backends
+/// transparently
+///
+/// Offers some more high level functionality compared to the lower level
+/// backend specific readers that this class wraps. In contrast to the lower
+/// level readers that usually return arbitrary FrameData, this interface class
+/// will return fully constructed Frames.
+///
```

Comment:
In addition it provides convenience methods to deal specifically with the event frame category...